internal/configs/version2/nginx-plus.transportserver.tmpl

```diff
@@ -29,4 +29,12 @@ server {
     {{ end }}
 
     proxy_pass {{ $s.ProxyPass }};
+
+    proxy_timeout {{ $s.ProxyTimeout }};
```

Comment:
would it make sense to also update https://github.com/nginxinc/kubernetes-ingress/blob/fb1f1cba932208c105f2ebc317ce011d799eaef5/internal/configs/version2/templates_test.go#L317 ?